docs/faq.md

```diff
@@ -241,3 +241,22 @@ grey_allOf(grey_sufficientlyVisible(),
 then all elements in the entire application will be checked for visibility, and finally one
 with a matching id will be selected. It's significantly faster to use the most targeted
 matchers first (typically `grey_accessibilityID` or `grey_accessibilityLabel`).
+
+**How do I find off screen elements?**
+
+EarlGrey requires elements to be visible (in the UI hierarchy) to perform automation.
+Just as a user would, scroll elements into view before interacting with them.
+
+```swift
+// Swift
+EarlGrey.selectElementWithMatcher(matcher)
+        .usingSearchAction(grey_scrollInDirection(.Down, 200),
+                           onElementWithMatcher:grey_kindOfClass(UITableView.self))
```

Comment:
you'd also need to add a performAction() or assert() call for this to actually do anything.